### PR TITLE
Release notes 6.36

### PR DESCRIPTION
**Pull request by vkostyukov**
Release notes for 6.36. Ping @finagle/csl.

### CODE REVIEW

File: source/blog/2016-07-08-release-notes-6-36.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+title: Finagle 6.36 Release Notes
+published: true
+post_author:
+  display_name: Vladimir Kostyukov
+  twitter: vkostyukov
+tags: Releases, Finagle, Util, Ostrich, Scrooge, TwitterServer
+---
+
+This is Finagle's first release only for Java 8 and Scala 2.11. Since previous release, the team
+has been focusing on Netty 4 migration as well as eliminating some technical debt.
```
Comment (kevinoliver) [inline source/blog/2016-07-08-release-notes-6-36.md]: on the Netty 4

File: source/blog/2016-07-08-release-notes-6-36.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+### Towards Netty 4
+
+The team is firing on all cylinders towards Netty 4 support in Finagle. This milestone brings
+*experimental* (not yet tested in production) Netty 4 support to Mux/ThriftMux, HTTP 1.1, and
```
Comment (kevinoliver) [inline source/blog/2016-07-08-release-notes-6-36.md]: what do you think of calling it beta instead? we have prod usage for http/1.1 and mux.

File: source/blog/2016-07-08-release-notes-6-36.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+SOCKS5 (proxy).
+
+In the meantime, we're half way through migrating MySQL, Memcached, and Redis protocol. Everything
```
Comment (kevinoliver) [inline source/blog/2016-07-08-release-notes-6-36.md]: s/half way/part way/

File: source/blog/2016-07-08-release-notes-6-36.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+goes quite smoothly (except for a couple of [shortcuts we had to take with Redis][redis-api]) and
```
Comment (kevinoliver) [inline source/blog/2016-07-08-release-notes-6-36.md]: s/goes quite/is going/

File: source/blog/2016-07-08-release-notes-6-36.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+### Killing the Tech Debt
+
+There are happened to be a tech debt sprint this milestone where we fixed up a plenty of broken
+windows. To mention a few: 
+
+- replace our own version of `NoStacktrace` with Scala's one
+- replace our own fork of JSR166e with the JDK8's one
+- clean up duplicated stats: `closechans`, `closed`, and `load`
+
+
```
Comment (kevinoliver) [inline source/blog/2016-07-08-release-notes-6-36.md]: it'd be good to call out the zipkin-core changes.

File: source/blog/2016-07-08-release-notes-6-36.md (hunk 2 above)
Comment (vkostyukov) [inline source/blog/2016-07-08-release-notes-6-36.md]: How about "brings **beta** (not yet widely used in production) Netty 4 support"?

File: source/blog/2016-07-08-release-notes-6-36.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+layout: post
+title: Finagle 6.36 Release Notes
+published: true
+post_author:
+  display_name: Vladimir Kostyukov
+  twitter: vkostyukov
+tags: Releases, Finagle, Util, Ostrich, Scrooge, TwitterServer
+---
+
+This is Finagle's first release only for Java 8 and Scala 2.11. Since previous release, the team
```
Comment (kevinoliver) [inline source/blog/2016-07-08-release-notes-6-36.md]: Since the previous

File: source/blog/2016-07-08-release-notes-6-36.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+This is Finagle's first release only for Java 8 and Scala 2.11. Since the previous release, the team
+has been focusing on the Netty 4 migration as well as eliminating some technical debt.
+
+### Towards Netty 4
+
+The team is firing on all cylinders towards Netty 4 support in Finagle. This milestone brings
+*beta* (not yet widely used in production) Netty 4 support to Mux/ThriftMux, HTTP 1.1, and
+SOCKS5 (proxy).
+
+In the meantime, we're part way through migrating MySQL, Memcached, and Redis protocol. Everything
```
Comment (mosesn) [inline source/blog/2016-07-08-release-notes-6-36.md]: s/migrating MySQL, Memcached, and Redis protocol/migrating the MySQL, Memcached, and Redis protocols

File: source/blog/2016-07-08-release-notes-6-36.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+is going quite smoothly (except for a couple of [shortcuts we had to take with Redis][redis-api]) and
+we're super positive about having an experimental Netty 4 support for those protocols in the
+next release.
+
+### Killing the Tech Debt
```
Comment (mosesn) [inline source/blog/2016-07-08-release-notes-6-36.md]: s/Killing the Tech Debt/Killing Tech Debt

File: source/blog/2016-07-08-release-notes-6-36.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+
+There are happened to be a tech debt sprint this milestone where we fixed up a plenty of broken
```
Comment (mosesn) [inline source/blog/2016-07-08-release-notes-6-36.md]: s/There are happened to be a/There was a
s/fixed up a plenty/fixed up plenty

File: source/blog/2016-07-08-release-notes-6-36.md (hunk 8 above)
Comment (jcrossley) [inline source/blog/2016-07-08-release-notes-6-36.md]: s/fixed up a plenty/fixed plenty

File: source/blog/2016-07-08-release-notes-6-36.md [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+windows. To mention a few: 
+
+- replace our own version of `NoStacktrace` with Scala's one
+- replace our own fork of JSR166e with the JDK8's one
```
Comment (jcrossley) [inline source/blog/2016-07-08-release-notes-6-36.md]: s/with the JDK8's one/ with the JDK8 version

File: source/blog/2016-07-08-release-notes-6-36.md [hunk 11]
```diff
@@ -0,0 +1,55 @@
...
+is going quite smoothly (except for a couple of [shortcuts we had to take with Redis][redis-api]) and
+we're super positive about having an experimental Netty 4 support for those protocols in the
+next release.
+
+### Killing the Tech Debt
+
+There are happened to be a tech debt sprint this milestone where we fixed up a plenty of broken
+windows. To mention a few: 
+
+- replace our own version of `NoStacktrace` with Scala's one
```
Comment (jcrossley) [inline source/blog/2016-07-08-release-notes-6-36.md]: s/with Scala's one/with Scala's version

File: source/blog/2016-07-08-release-notes-6-36.md [hunk 12]
```diff
@@ -0,0 +1,55 @@
...
+- replace our own fork of JSR166e with the JDK8's one
+- clean up duplicated stats: `closechans`, `closed`, and `load`
+- finagle-zipkin is now split into two packages: finagle-zipkin (implementation) and
+  finagle-zipkin-core (core data types and interfaces)
+
+### Side Effects are Side Effects
+
+There is one runtime change we think it's worth to highlight here. We changed the behaviour of
```
Comment (jcrossley) [inline source/blog/2016-07-08-release-notes-6-36.md]: s/it's worth to highlight/it's worth highlighting

File: source/blog/2016-07-08-release-notes-6-36.md [hunk 13]
```diff
@@ -0,0 +1,55 @@
...
+`.onSuccess`/`.onFailure` combinators so that they don't fail a request if the passed function
+throws an exception. Please, consider refactoring if you're application has been relying on that
```
Comment (jcrossley) [inline source/blog/2016-07-08-release-notes-6-36.md]: s/you're/your

**Pull request merged**